tests/ExcelProvider.Tests/ExcelProvider.Tests.fs

```diff
@@ -323,7 +326,7 @@ let ``Cannot create a type referring to a non-existant sheet at runtime``() =
     //| Choice2Of2 v -> printfn "2 %A" v
     //printfn "%A" diag
     let dstr = diag[0].ToString()
-    dstr.Substring(dstr.IndexOf "typecheck") |> should equal "typecheck error The type provider 'FSharp.Interop.Excel.ExcelProvider.ProviderImplementation+ExcelProvider' reported an error: Sheet [C] does not exist."
+    dstr.Substring(dstr.IndexOf "typecheck") |> should equal "typecheck error The type provider 'FSharp.Interop.Excel.ExcelProvider.ProviderImplementation+ExcelProvider' reported an error: ExcelProvider: Sheet [C] does not exist."
```

Comment:
I found this when running the tests locally. Commit 355aa8c added "ExcelProvider: " in front of all error strings, and this test just wasn't updated to reflect that yet.